consensus/XDPoS/engines/engine_v2/engine.go

```diff
@@ -164,7 +164,7 @@ func (x *XDPoS_v2) Initial(chain consensus.ChainReader, header *types.Header) er
 }
 
 func (x *XDPoS_v2) initial(chain consensus.ChainReader, header *types.Header) error {
-	log.Info("[initial] initial v2 related parameters")
+	log.Warn("[initial] initial v2 related parameters")
```

Comment:
why is this a warn? doesn't look like an error msg to me

Reply:
This is not Warn, however most of owner only shows warn and error log, it's hard to know whether they really run this function or not. This is like critical info.